server/server_read_report_metainfo_test.go

```diff
@@ -0,0 +1,149 @@
+// Copyright 2021 Red Hat, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package server_test
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/RedHatInsights/insights-results-aggregator-data/testdata"
+
+	"github.com/RedHatInsights/insights-results-aggregator/server"
+	"github.com/RedHatInsights/insights-results-aggregator/tests/helpers"
+)
+
+func TestReadReportMetainfoForClusterNonIntOrgID(t *testing.T) {
```

Comment:
Some of these tests can be parameterized by the endpoint, first endpoint args, status code and body. This way the code is not duplicated. WDYT? Not a big deal btw.

Reply:
can in theory. TBH for tests (just tests) I prefer to be more talkative that to be "smarter" (we know where it ends - in the RESTAPI helpers nobody is able to tell the parameters ;)

Reply:
sure, it's a valid trade over :)